cro-clib/src/fee.rs

```diff
@@ -41,6 +41,25 @@ pub unsafe extern "C" fn cro_estimate_fee(fee_ptr: CroFeePtr, tx_payload_size: u
     let coin_value: u64 = fee_value.to_coin().into();
     coin_value
 }
+
+/// estimate fee after encryption
+/// tx_payload_size: in bytes
+#[no_mangle]
+/// # Safety
+pub unsafe extern "C" fn cro_estimate_fee_after_encrypt(
```

Comment:
not sure if used?

Reply:
if there is fee, it will be used.
because user need to compute fee, decide the amount to return to the sender.